### ISSUE TITLE
Feature requests: China and United States drill-down, on geolocalization

### ISSUE THREAD

**Issue by par7133** [allinurl/goaccess#2722]
Hello,

Firstly I want to take the time to thank you for your great job as 5 Mode is starting to use Goaccess just now and I'm happy very about it.

I'm here to ask as per subject the possibility to drill down in detail contries like United States and China that look like my user favorite locations too (having no possibility to ask him to stop in one location..).

Sorry if my requet is repeated,

-Dan

**Comment by allinurl** [comment]
When you mention "drill down," are you talking about the capability to examine specific cities within those countries?

**Comment by par7133** [comment]
Yes, exactly. And if not States examination is possible for United States.

-Dan

Sep 26, 2024 02:07:43 Gerardo O. ***@***.***>:

**Comment by allinurl** [comment]
This has been implemented in the [latest version](https://goaccess.io/). v1.10.1. e.g., 

<img width="712" height="530" alt="Image" src="https://github.com/user-attachments/assets/edfe57b6-5684-4f00-8125-74bb42ae47d4" />

**Comment by dtmf7133** [comment]
Thx, your effort was appreciated..

Dan

Feb 16, 2026 02:45:27 Gerardo O. ***@***.***>:
